### PR TITLE
🤖  Erro ao criar editor no Windows

### PR DESCRIPTION
**Pull request by marcelofarias**
[Changes reviewed on CodeStream](https://pd-api.codestream.us/r/XvOAbMlJ3Tpk0aJ7/4f7p9eBMR6mx5RnSFfC2CA?src=GitHub) by marcia on Jun 24, 2020

### CODE REVIEW

File: editor.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ if (code.length === 0) {
 
 this.focusInput();
 
-let filePath = editor.getPath();
+let filePath = editor.getPath().normalizePath();
```
Comment (marcelofarias) [inline editor.js:10]: foo

Comment (marcelofarias) [inline editor.js:10]: a comment

Comment (marcelofarias) [inline editor.js:10]: a comment

Comment (marcelofarias) [inline editor.js:10]: fooz

Comment (marcelofarias) [inline editor.js:10]: test 234

Comment (marcelofarias) [inline editor.js:10]: asdfds

Comment (marcelofarias) [inline editor.js:10]: asdffsd

Comment (marcelofarias) [inline editor.js:10]: asdfasdf

Comment (marcelofarias) [inline editor.js:10]: asdfsdaf

Comment (marcelofarias) [inline editor.js:10]: asdfasdf

Comment (marcelofarias) [inline editor.js:10]: saasfdsdf

Comment (marcelofarias) [inline editor.js:10]: asfasdfafdsaf

Comment (eric-j) [inline editor.js:10]: test

Comment (eric-j) [inline editor.js:10]: test

Comment (eric-j) [inline editor.js:10]: test2

Comment (planteater) [inline editor.js:10]: this from dave via CS

Comment (planteater) [inline editor.js:10]: this is from dave via GH

**Pull request closed without merging**